posthog/models/kafka_engine_dlq/sql.py

```diff
@@ -0,0 +1,21 @@
+KAFKA_ENGINE_DLQ_BASE_SQL = """
```

Comment:
Let's add a comment to the top of this file outlining what this does and what the motivation for this is.